Jenkinsfile

```diff
@@ -34,8 +37,11 @@ properties([
     string(name: 'REPEAT_TESTS',
            defaultValue: '1',
            description: 'How many times to repeat each test.'),
+    string(name: 'RETRIES',
+           defaultValue: '0',
+           description: 'How many times to retry each failed test'),
     string(name: 'ROBOTEST_VERSION',
-           defaultValue: 'stable-gce',
+           defaultValue: 'uid-gid',
```

Comment:
I released 2.1.1 (compatible with 5.0.x) today.  It is available if you want to pick it up.